### ISSUE TITLE
Bimview 3d not work

### ISSUE THREAD

**Issue by db509** [opensourceBIM/BIMserver#1137]
Goodmorning everyone,
I recently started testing bimserver and I'm trying to install it on centos 7. The installation of the latest available version is successful but after loading an IFC project I can't see it in 3D in Bimviews.
I have seen that in issue 1090 there is talk of incorrect version of plugins. Could you show me a working combination of servers and plugins?
I thank you in advance for your valuable support.
Greetings.

Dario

**Comment by dzhxie** [comment]
I have the same issue and did not find the solution. I am using the latest version with IFCShell plugin installed.

**Comment by tciles** [comment]
I had the same issue, i tried with centos8 and debian10 and this issue disappeared

**Comment by db509** [comment]
Hi tciles, very thanks for your message. This was the solution for me. I've now installed version 1.5.182 on Centos 8 and it works!
I've used Open JDK 11 and Tomcat 9.

On CentOS 7 and tomcat 8.5 after consulting log file the problem was: ERROR [GeometryRunner] o.b.g.StreamingGeometryGenerator [GeometryRunner.java:847] 

With the new configuration as installed with the wizard only two warning: 

WARN [pool-1-thread-1] o.b.g.StreamingGeometryGenerator [StreamingGeometryGenerator.java:661] [ARCHITETTONICO.ifc] Number of representations skipped:
WARN [pool-1-thread-1] o.b.g.StreamingGeometryGenerator [StreamingGeometryGenerator.java:663] 	FootPrint: 1

Very Very thanks for the support.
Dario
![project loaded 1 5 182](https://user-images.githubusercontent.com/64728365/83541213-a2997780-a4f9-11ea-8239-803a29b2b281.PNG)
